packages/runner/src/types/tasks.ts

```diff
@@ -189,9 +189,9 @@ export type SuiteAPI<ExtraContext = {}> = ChainableSuiteAPI<ExtraContext> & {
   runIf(condition: any): ChainableSuiteAPI<ExtraContext>
 }
 
-export type HookListener<T extends any[], Return = void> = (...args: T) => Awaitable<Return>
+export type HookListener<T extends any[], Return = unknown> = (...args: T) => Awaitable<Return>
 
-export type HookCleanupCallback = (() => Awaitable<unknown>) | void
+export type HookCleanupCallback = (() => Awaitable<unknown>) | unknown
```

Comment:
I am not sure about this one. `unknown` absorbs the callback definition, so it's no longer visible that you need to return a function.

Reply:
What would you suggest as an alternative?